src/vcf/record.cpp

```diff
@@ -704,7 +776,7 @@ namespace ebi
         }
         if(!values.empty()) {
             if (values.front() == MISSING_VALUE) { return; } // No need to check missing data
-        }
+        } //TODO, if the 1st one is . then check stops; is that correct? svclaim=.,DJ worked!
```

Comment:
This is not correct indeed: 
```
SVCLAIM=.
```
Is correct because missing value applies to all value
```
SVCLAIM=.,DJ
```
Needs to be checked for cardinality